src/components/views/messages/RoomAvatarEvent.tsx

```diff
@@ -70,7 +70,7 @@ export default class RoomAvatarEvent extends React.Component<IProps> {
                                 className="mx_RoomAvatarEvent_avatar"
                                 onClick={this.onAvatarClick}
                             >
-                                <RoomAvatar size="14px" oobData={oobData} />
+                                <RoomAvatar room={room ?? undefined} size="14px" oobData={oobData} />
```

Comment:
We include the room here so that the media is trusted -- usually a RoomAvatar without a room to get the membership from is untrusted as we can't confirm if we're invited (e.g. 3pid invites).

Nearly all other usages of <RoomAvatar> include a room, this was one of the few occurrences.